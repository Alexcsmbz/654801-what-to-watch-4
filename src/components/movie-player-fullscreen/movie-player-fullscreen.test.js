import MoviePlayerFullscreen from './movie-player-fullscreen.tsx';

const movie = {
  name: `Fantastic Beasts: The Crimes of Grindelwald`,
  thumbnail: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  genre: `Comedies`,
  releaseDate: `2014`,
  promo: `promo`,
  poster: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  previewMp4: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  previewWebm: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`
};

describe(`MoviePlayerFullscreen snapshot test`, () => {
  it(`MoviePlayerFullscreen should render fullscreen movie player`, () => {
    const tree = renderer.create(
        <MoviePlayerFullscreen
          setIsFullscreen={() => {}}
          movie={movie}
          isFullscreen={true}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
