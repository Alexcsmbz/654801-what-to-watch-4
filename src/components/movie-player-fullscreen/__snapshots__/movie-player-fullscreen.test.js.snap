// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePlayerFullscreen snapshot test MoviePlayerFullscreen should render fullscreen movie player 1`] = `
<video
  controls={true}
  height="175"
  width="280"
>
  <source
    type="video/mp4"
  />
</video>
`;
